deepchem/models/torch_models/cgcnn.py

```diff
@@ -0,0 +1,337 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from deepchem.models.torch_models.torch_model import TorchModel
+
+
+class CGCNNLayer(nn.Module):
```

Comment:
Can you add these classes to `deepchem/docs/models.rst`? We should add a new section in the docs file for PyTorch models and add these there.

Reply:
OK! But, could I make the another PR...?

Reply:
Sounds good!